doc/README.md

```diff
@@ -12,8 +12,8 @@ Pacemaker has multiple forms of documentation:
     * Pacemaker Remote: Configuration and walk-throughs for extended
       clusters
 
-  The source for these is kept beneath this directory. Generated versions
-  are available online in epub, PDF, and HTML format at:
+  The source for these is kept beneath in this directory's sphinx subdirectory.
```

Comment:
I think you could just lose the word "beneath" here and it would make more sense.

Reply:
Whoops, yep